src/transformers/modeling_tf_utils.py

```diff
@@ -515,7 +515,7 @@ def call(self, inputs, training=False):
             cls_index = inputs[1] if len(inputs) > 1 else None
             assert len(inputs) <= 2, "Too many inputs."
         else:
-            input_ids = inputs.get("input_ids")
+            hidden_states = inputs.get("hidden_states")
```

Comment:
I made this change because:

- the docstring suggests the input argument should contain hidden states; and
- without this change, if the code goes through this `else` branch, it'll fail soon thereafter because the `hidden_states` variable isn't defined.

This is a wild guess.